build/KoreBuild.sh

```diff
@@ -14,7 +15,13 @@ __exec() {
 
     local cmdname=$(basename $cmd)
     echo -e "${CYAN}> $cmdname $@${RESET}"
-    $cmd "$@"
+
+    if [ -z "${TRAVIS}" ]; then
+        $cmd "$@"
+    else
+        # Work around https://github.com/Microsoft/msbuild/issues/1792
+        $cmd "$@" | tee /dev/null
```

Comment:
:ok:

Strange that didn't happen in my local testing. For example I ran `./build.sh /t:fail` after `export TRAVIS=true` and the script noticed the non-`0` exit code.